autogen.sh

```diff
@@ -601,6 +601,27 @@ else
     fi
 fi
 
+########################################################################
+## Check for Python 3
+########################################################################
+
+echo_n "Checking for Python 3... "
+PYTHON=
+if test 3 = `python -c 'import sys; print(sys.version_info[0])'`; then
+    PYTHON=python
+fi
+
+if test -z "$PYTHON" -a 3 = `python3 -c 'import sys; print(sys.version_info[0])'`; then
+    PYTHON=python3
```

Comment:
OK, now that I look at this closer I see that we are able to support both `python` and `python3`, which was most of my concern. I don't think that mess will be sorted out for some time.